From 866a09661de3c83b227dbc0ce49da25976229577 Mon Sep 17 00:00:00 2001
From: chewitt <github@chrishewitt.net>
Date: Sat, 18 May 2019 08:11:40 +0000
Subject: [PATCH 7/7] arm64: dts: meson-g12a-x96-max: set AMLG12 audio card
 name

The audio card name needs to match a generic alsa .conf shared by all
G12A/B devices so change the name to AMLG12.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-g12a-x96-max.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12a-x96-max.dts b/arch/arm64/boot/dts/amlogic/meson-g12a-x96-max.dts
index ef027001b9e5..f96d2aa31ff7 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12a-x96-max.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-g12a-x96-max.dts
@@ -133,7 +133,7 @@
 
 	sound {
 		compatible = "amlogic,axg-sound-card";
-		model = "G12A-X96-MAX";
+		model = "AMLG12";
 		audio-aux-devs = <&tdmout_a>, <&tdmout_b>, <&tdmout_c>;
 		audio-routing = "TDMOUT_A IN 0", "FRDDR_A OUT 0",
 				"TDMOUT_A IN 1", "FRDDR_B OUT 0",
-- 
2.17.1

